pom.xml

```diff
@@ -445,7 +445,7 @@
             <groupId>org.owasp</groupId>
             <artifactId>dependency-check-maven</artifactId>
             <configuration>
-              <failBuildOnCVSS>8</failBuildOnCVSS>
+              <failOnError>false</failOnError>
```

Comment:
In general this is the config we need. However there are two ways to use the plugin:
1. as a single check (mvn dependency:check) [* Example 7]
2. as a report plugin for the maven site [* Example 2]

The site (2.) should always build. In the resulting HTML you can then see failing tests, vulnerabilities in depndencies, etc. whereas for (1.) the build-breaker still makes sense. Can you hence move the new config (failOnError) to the reporting config and keep the existing config for the build-section invocation (and CLI)?

(*) https://jeremylong.github.io/DependencyCheck/dependency-check-maven/